src/main/features/claim/routes/defendant-sole-trader-details.ts

```diff
@@ -0,0 +1,32 @@
+import * as express from 'express'
+
+import { Paths } from 'claim/paths'
+
+import { Form } from 'forms/form'
+import { FormValidator } from 'forms/validation/formValidator'
+import { SoleTraderDetails } from 'forms/models/soleTraderDetails'
+
+import { ClaimDraftMiddleware } from 'claim/draft/claimDraftMiddleware'
+import { ErrorHandling } from 'common/errorHandling'
+
+function renderView (form: Form<SoleTraderDetails>, res: express.Response): void {
+  res.render(Paths.defendantSoleTraderOrSelfEmployedDetailsPage.associatedView, { form: form })
+}
+
+export default express.Router()
+  .get(Paths.defendantSoleTraderOrSelfEmployedDetailsPage.uri, (req: express.Request, res: express.Response) => {
```

Comment:
Having separate routes without any extra control might lead that lets say defendant initiated as Individual is displayed on details page dedicated for example organisation.